egs/csj/ASR/local/compute_fbank_csj.py

```diff
@@ -0,0 +1,140 @@
+import argparse
```

Comment:
Could you follow
https://k2-fsa.github.io/icefall/contributing/code-style.html
to fix your code style issues?

Reply:
Thanks for the link. I have just done that. 

There is one file where I declared a long list of variables. It seems black and flake8 can't agree on whether to add a newline.

Reply:
To by-pass the check for some line, you can append ` # noqa` at the end of that line. For instance
```python3
this_is_a_very_long_name = 1  # noqa
```